tests/bsim/bluetooth/host/att/mtu_update/src/main_central.c

```diff
@@ -14,15 +14,15 @@
 
 #include <zephyr/bluetooth/gatt.h>
 
-#include "common.h"
-#include "time_machine.h"
+#include "babblekit/testcase.h"
+#include "babblekit/flags.h"
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_bsim_mtu_update, LOG_LEVEL_DBG);
 
 extern void run_central_sample(void *cb);
 
-CREATE_FLAG(flag_notification_received);
+static DEFINE_FLAG(flag_notification_received);
```

Comment:
Several places in Zephyr we have a static and non-static version of macros, so we could consider a `DEFINE_STATIC_FLAG` or `DEFINE_FLAG_STATIC`. 

Not a requirement, but something to consider

Reply:
Good suggestion, added!